app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.kt

```diff
@@ -88,6 +90,11 @@ class ZimManageActivity : BaseActivity() {
     searchItem?.isVisible = position == 1
     languageItem?.isVisible = position == 1
     getZimItem?.isVisible = position == 0
+    if (position == 1) {
+      val fragment: Fragment?
+      fragment = ZimFileSelectFragment()
```

Comment:
this creates a fragment, you want to retrieve the existing instance from the viewpager's adapter.

You also probably want to do this in another pageChangeListener